lib/active_storage/attached/one.rb

```diff
@@ -1,25 +1,39 @@
+# Representation of a single attachment to a model.
 class ActiveStorage::Attached::One < ActiveStorage::Attached
   delegate_missing_to :attachment
 
+  # Returns the associated attachment record.
```

Comment:
Let's talk about how you usually don't need to reference it directly as delegate_missing will send messages there.